packages/web-service/src/pages/applications/applications.js

```diff
@@ -0,0 +1,22 @@
+import pageRoute from '../../routes/page-route.js'
+import { APPLICATIONS } from '../../uris.js'
+import fetch from 'node-fetch'
+
+const tmpUserId = '579d4e05-e9d8-472f-a9a9-3fc52234d88b'
+
+export async function fetchApplications (userId) {
+  try {
+    const response = await fetch(`${process.env.API_HOST}:${process.env.API_PORT}/user/${userId}/applications`)
```

Comment:
Consider using URL which will validate a correct URL string.

new URL(`http://${process.env.API_HOST}:${process.env.API_PORT}/user/${userId}/applications`)

Also consider using the pattern in the guidance for fetch in https://www.npmjs.com/package/node-fetch

The 3xx-5xx are not exceptions so it is recommended to use a function like


const checkStatus = response => {
	if (response.ok) {
		// response.status >= 200 && response.status < 300
		return response;
	} else {
		throw new HTTPResponseError(response);
	}
}

(See the link for details).

Note we will need to actually create an error page for 400 and 500 errors at some pointy which we will redirect to. For the moment maybe just redirect to a holding page.

Consider also abstracting out all API operations as they will likely follow exactly the same pattern.